core/src/main/java/com/orange/ouds/core/component/OudsControlItem.kt

```diff
@@ -269,6 +276,38 @@ private fun LeadingTrailingBox(content: @Composable () -> Unit) {
     }
 }
 
+@Composable
+private fun ErrorDescriptionText(text: String) {
+    with(OudsTheme.componentsTokens.controlItem) {
+        Text(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = spacePaddingBlockTopErrorText.value)
+                .padding(horizontal = spacePaddingInline.value)
+                .semantics {
```

Comment:
TalkBack read the error description twice, this is the same bug we fixed for the text input (#913).

We should use `clearAndSetSemantics` to fix it.